default.vcl

```diff
@@ -224,11 +229,13 @@ sub vcl_recv {
 
 
     // allow notifications-push health and gtg checks to pass without requiring auth
-    if ((req.url ~ "^\/__(list-|page-)?notifications-push\/__health.*$") || (req.url ~ "^\/__(list-|page-)?notifications-push\/__gtg.*$")) {
+    if ((req.url ~ "^\/__(annotations-|list-|page-)?notifications-push\/__health.*$") || (req.url ~ "^\/__(annotations-|list-|page-)?notifications-push\/__gtg.*$")) {
```

Comment:
**Question:** Shouldn't this be `annotation-` instead of `annotations-`

Reply:
It should, thanks for spotting it!